### ISSUE TITLE
manufacturer-logo cms element: Cannot read properties of undefined (reading 'value')

### ISSUE THREAD

**Issue by ahausamm** [shopware/platform#13620]
### Shopware Version

6.7.4.2

### Affected area / extension

Platform(Default)

### Actual behaviour

Adding a product-heading block to the shopping experiences page leads into a runtime error "Cannot read properties of undefined (reading 'value')". If i got it right the root cause seems to be in the manufacturer-logo element which itself extends from image element. The default config is missing the new fields **ariaLabel** and **fetchPriorityHigh** which were added in the following PRs:

- https://github.com/shopware/shopware/pull/13070
- https://github.com/shopware/shopware/pull/12865

Not sure if there are any other occasions.

### Expected behaviour

- No more runtime error
- Config is accessible

### How to reproduce

1. Opening a shopping experience page in designer
2. Add block "Commerce / Product Heading"
3. Check console for runtime error and see that config modal for image is empty

**Comment by MartinKrzykawski** [comment]
Hello @ahausamm, thanks for your inquiry!

The issue is that the CMS commerce element “Product name & manufacturer logo” does not provide any real added value when used within a non-commerce layout. This element is designed to work with product data and therefore functions correctly when used, for example, on a product detail page.

In non-commerce layouts, however, no product context is available. As a result, the element only consists of an empty text field and an image element and must be manually configured before it can be saved. This behavior can be confusing, as the element may appear to be broken even though it is technically working as intended.

To address this, I’ve created a ticket to improve the workflow and document this behavior more clearly to avoid confusion in the future:
https://github.com/shopware/shopware/issues/14125
